certbot/certbot/_internal/client.py

```diff
@@ -557,11 +557,6 @@ def enhance_config(self, domains, chain_path, ask_redirect=True):
                 if ask_redirect:
                     if config_name == "redirect" and config_value is None:
                         config_value = enhancements.ask(enhancement_name)
-                        if not config_value:
```

Comment:
I think we should leave this code. My understanding is the only time we will enter this branch is if the result of `enhancements.ask` is to not provide a redirect which I think is only possible if the user interactively asks for no redirect.

If we do this, we should probably keep the test.

Reply:
Oh, good call, agreed.